lib/node_modules/@stdlib/math/base/special/minmax/lib/native.js

```diff
@@ -0,0 +1,55 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var addon = require( './../src/addon.node' );
+
+
+// MAIN //
+
+/**
+* Returns the minimum and maximum value, ignoring NaN.
+*
+* @private
+* @param {number} x - first number
+* @param {number} y - second number
+* @returns {number} minimum and maximum value
+*
+* @example
+* var v = minmax( 3.14, 4.2 );
+* // returns [ 3.14, 4.2 ]
+*
+* @example
+* var v = minmax( 4.14, NaN );
+* // returns [ NaN, NaN ]
+*
+* @example
+* var v = minmax( NaN, NaN );
+* // returns [ NaN, NaN ]
+*/
+function minmax( x, y ) {
+	return addon( x, y );
+}
```

Comment:
I think your `C` implementation should be similar to how its done in the `assign.js`.
```suggestion
function minmax( x, y ) {
        var Out;
        Out = [ 0.0, 0.0 ];
        addon( x, y, Out, 1, 0 );
	return Out;
}
```

Reply:
@headlessNode I have implemented that functionality. Please review my pr.